### ISSUE TITLE
NTR: Lens capsule rupture (Eye structure & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7989]
New term request Lens capsule rupture (elthat_150106170921):   UMLS: 3671783

**Comment by pnrobinson** [comment]
@MickeySegal
Does this refer to the complication of cataract surgery, e.g., https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2377380/ ?

**Comment by MickeySegal** [comment]
It was for the occasional abnormality listed in Smith's "Recognizable patterns of human malformation" for Marshall syndrome "spontaneous rupture of lens capsule".  We didn't use the word spontaneous because it presumably happens after some trivial trauma to the eye, but I've now added the explanatory material "Not from major trauma such as surgery".

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000097
- label: Spontaneous lens capsule rupture
- parents: [Abnormal lens morphology (HP:0000517)](https://hpo.jax.org/app/browse/term/HP:0000517)
- pmid: PMID:35198824
- def: Breakage of the capsule of the lens. This is most often observed as a complication of lens surgery, but this term refers to lens capsule rupture not related to surgery or major trauma.
- comment: Spontaneous lens capsule rupture has been observed with Marshall syndrome.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7989
- synonyms:
